astropy/timeseries/periodograms/lombscargle/implementations/fast_impl.py

```diff
@@ -39,9 +40,12 @@ def lombscargle_fast(
     use_fft : bool (default=True)
         If True, then use the Press & Rybicki O[NlogN] algorithm to compute
         the result. Otherwise, use a slower O[N^2] algorithm
+    prefer_lra : bool
+        if True and SciPy is available, use the more accurate Low Rank Approximation by Ruiz-Antolin and Townsend.
+        If False or SciPy is not available use Press & Rybicki's Lagrangian extirpolation instead.
```

Comment:
I would consider it surprising that I, as a user, explicitly request LRA be used and `astropy` silently ignoring it on the ground that I don't have `scipy` installed. I would recommend making it a hard error instead.